core/Constants.php

```diff
@@ -87,6 +87,25 @@ function stripslashes_recursively(&$array) {
 }
 
 /**
+ * Check if a given ip is in a network
+ * @param  string $ip    IP to check in IPV4 format eg. 127.0.0.1
+ * @param  string $range IP/CIDR netmask eg. 127.0.0.0/24, also 127.0.0.1 is accepted and /32 assumed
+ * @return boolean true if the ip is in this range / false if not.
+ */
+function ip_in_range( $ip, $range ) {
+	if ( strpos( $range, '/' ) == false ) {
```

Comment:
If there's no slash, then it's not a range, it's a single IP, right? so we can do something like: `$ip == $range`